packages/chain-adapters/src/ethereum/EthereumChainAdapter.ts

```diff
@@ -308,14 +333,16 @@ export class ChainAdapter implements IChainAdapter<ChainTypes.Ethereum> {
     await this.providers.ws.subscribeTxs(
       subscriptionId,
       { topic: 'txs', addresses: [address] },
-      (msg) => {
-        const transfers = msg.transfers.map<chainAdapters.TxTransfer>((transfer) => ({
-          caip19: transfer.caip19,
-          from: transfer.from,
-          to: transfer.to,
-          type: getType(transfer.type),
-          value: transfer.totalValue
-        }))
+      (msg: chainAdapters.SubscribeTxsMessage<ChainTypes.Ethereum>) => {
```

Comment:
These changes to subscribeTxs _shouldn"t_ be required I wouldn't think (both eth and btc)